Cod3rsGrowth.Web/Program.cs

```diff
@@ -1,15 +1,15 @@
+using Cod3rsGrowth.Infra.Repositorio;
+
 var builder = WebApplication.CreateBuilder(args);
 
-// Add services to the container.
 
 builder.Services.AddControllers();
-// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddScoped<IRepositorio, RepositorioComLinq2Db>();
```

Comment:
Exatamente isso, quando for mudar de repositório, sempre altera aqui, usando a injeção